.github/workflows/gh-ci-cron.yaml

```diff
@@ -223,7 +223,7 @@ jobs:
     strategy:
         fail-fast: false
         matrix:
-          os: [ubuntu, macos]
+          os: [ubuntu, macos, macos-14]
```

Comment:
This won't work I think - you'd need to temporarily enable cron CI, but this is doing a full deps install via conda and it'll fail because there are many missing deps for arm64-osx. Just doing the pypi-only check would be fine here.

Reply:
Yeah, I added those changes but then realized they are not tested... 

Full install does not work indeed (`hole2`, `distopia`, ...). I agree that PyPI check would be enough here, and minimizes the chances of errors.

Reply:
you can just change the header of the file to trigger on pull_request and it'll run here

Reply:
Smart! =)

Reply:
I forgot about this trick, thanks.